gtwrap/interface_parser/tokens.py

```diff
@@ -46,6 +46,7 @@
         "#include",
     ],
 )
+ENUM = Keyword("enum") ^ Keyword("enum class") ^ Keyword("enum struct")
```

Comment:
pybind allows you to `.export_values()` or not depending on whether the enum is strongly typed or not.  Probably this behavior should be kept?

e.g. expected behavior:
```c++
class Foo {
  enum Enum1 { a1, a2, a3 };
  enum struct Enum2 { b1, b2, b3 };
};
```
```python
Foo.a1        # ok
Foo.Enum1.a1  # not ok
Foo.b1        # not ok
Foo.Enum2.b1  # ok
```

Reply:
This doesn't seem like a good idea. Why would you use the enum value as a class variable?

Reply:
Sorry, would you mind clarifying?  I'm not sure what you mean - this is how enum's are normally scoped unless you use the `enum struct` or `enum class` keywords?  Unless you're proposing to delete the `Keyword("enum")` and `.export_values()` to force c++11 strongly typed enums, which would also be acceptable to me?

To clarify what I mean by the `Foo.a1` example, in c++, the value `a1` is accessed by `Foo::a1`, not `Foo::Enum1::a1`, so I assumed that python should have the same scoping.

Reply:
Yeah but when you're reading python code, would `Foo.a1` make sense or would `Foo.Enum1.a1`?

Reply:
I think xx.enum1.a1 is more Pythonic, the C++ enum thing is more like legacy...

Reply:
Ok, great!  so then just delete `.export_values` down below :)
I think we're all arguing the same thing: that `Foo.b1` should _not_ be allowed.  In the current implementation, `Foo.b1` _is_ allowed, because `.export_values` is being used even for strongly typed enums.  I felt that perhaps it was ok to leave the current behavior where `Foo.a1` is allowed in addition to `Foo.Enum1.a1` (but _not_ `Foo.b1`) since c++ syntax allows this, but I'm less concerned about that than I am about _dis_allowing `Foo.b1`, which requires deleting `.export_values();`.

Reply:
Done